### PR TITLE
Add support for retreiving level integer value

### PR DESCRIPTION
**Pull request by jsumners**
Defines a getter on the prototype with a noop setter.

### CODE REVIEW

File: pino.js [hunk 1]
```diff
@@ -84,6 +84,13 @@ Object.defineProperty(Pino.prototype, 'level', {
   set: Pino.prototype._setLevel
 })
 
+Object.defineProperty(Pino.prototype, 'levelInteger', {
+  get: function getInt () {
+    return levels[this._getLevel()]
+  },
+  set: function setInt () {}
```
Comment (mcollina) [inline pino.js]: This `set` is useless.

**Comment by satazor** [comment]
You missed updating the README.

File: README.md [hunk 2]
```diff
@@ -143,6 +143,7 @@ Into this:
   * <a href="#info"><code>logger.<b>info()</b></code></a>
   * <a href="#debug"><code>logger.<b>debug()</b></code></a>
   * <a href="#trace"><code>logger.<b>trace()</b></code></a>
+  * * <a href="#levelInteger"><code>logger.<b>levelInteger</b></code></a>
```
Comment (mcollina) [inline README.md]: there is a double `*`.

**Comment by mcollina** [comment]
Can you please merge/rebase it with master?

**Comment by jsumners** [comment]
Sorry, I forgot to pull in those changes.

**Comment by davidmarkclements** [comment]
should this be closed in light of #39 ?

**Comment by jsumners** [comment]
GitHub should have deleted this since the other side doesn't exist any longer.

**Pull request closed without merging**